spec/forms/triage_assistance_form_spec.rb

```diff
@@ -18,8 +18,7 @@
       let(:params) do
         {
           assistance_in_person: "no",
-          assistance_chat: "yes",
```

Comment:
This removes the one `yes` that we were saving into the model. I'd rather save at least 1 `yes`. Not a blocker.